arbnode/execution/sequencer.go

```diff
@@ -268,11 +268,16 @@ func (c nonceFailureCache) Contains(err NonceError) bool {
 }
 
 func (c nonceFailureCache) Add(err NonceError, queueItem txQueueItem) {
```

Comment:
It's called this because it's a cache of "nonce failure"s. It's not suitable to handle other types of failed transactions, as it only accepts NonceErrors as keys.

Reply:
Maybe an alternative name would be "wrongNonceTxCache"?